containers/ecr-viewer/src/app/components/LibrarySearch.tsx

```diff
@@ -17,19 +16,8 @@ interface LibrarySearchProps {
  * @returns - Search bar component for the eCR Library
  */
 const LibrarySearch = ({ className, textBoxClassName }: LibrarySearchProps) => {
-  const router = useRouter();
-  const pathname = usePathname();
-  const searchParams = useSearchParams();
+  const { searchParams, updateQueryParam, pushQueryUpdate } = useQueryParam();
```

Comment:
this was a side quest refactor when I was trying to figure out some odd caching-related-behavior. But I think it's cleaner, so I left it